spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
Be wary of generated files, they tend to use single quotation marks.
It's caught me out a few times too!